src/20_Components/amp-access-laterpay.html

```diff
@@ -0,0 +1,73 @@
+<!--
+  #### Introduction
+
+  AMP Access LaterPay allows publishers to easily integrate with the micropayments platform [LaterPay](https://www.laterpay.net). It is based on, and requires [AMP Access](https://www.ampproject.org/docs/reference/components/amp-access).
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-access-laterpay/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+  <!-- #### Setup -->
+  <!-- Import the `amp-access` component in the header. -->
+  <script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js"></script>
+  <!-- `amp-access` requires `amp-analytics` to be imported as well. -->
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <!-- And of course the `amp-access-laterpay` component. -->
+  <script async custom-element="amp-access-laterpay" src="https://cdn.ampproject.org/v0/amp-access-laterpay-0.1.js"></script>
+
+  <!-- Add the configuration which specifies the vendor, and vendor specific configuration for `amp-access` -->
+  <script id="amp-access" type="application/json">
+  {
+    "vendor": "laterpay",
+    "laterpay": {
+      "articleTitleSelector": ".preview > h3"
+    }
+  }
+  </script>
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+
+  <!-- This section is visible to all users. No special markup is needed in this case.-->
+  <section class="preview">
+    <h3>The Article</h3>
+    <p>
+      This is a preview of the article.
+    </p>
+  </section>
+
+  <!-- This section is only shown if the value of the `access` and `error` attributes 
+  on the response for the authorization requests both evaluate to `FALSE`-->
```

Comment:
Maybe add that the later pay dialog will be shown here. Are the `id` and `class` predefined?